src/lib/source-file-converter.ts

```diff
@@ -132,6 +140,49 @@ export class SourceFileConverter {
 
     const arraySymbol = symbols.filter((s) => s.name === 'Array')[0];
     this._arrayMapSymbol = arraySymbol.members!.get('map' as any)!;
+
+    // Get original html, property, and customElement symbols.
+    this.htmlSymbol = this.getExportOfModule('lit-html', 'html');
+    this.propertySymbol = this.getExportOfModule('lit-element/lib/decorators', 'property');
+    this.customElementSymbol = this.getExportOfModule('lit-element/lib/decorators', 'customElement');
+  }
+
+  /**
+   * Return the symbol named `symbolName` exported from the module named `moduleName`.
+   * If the module or symbol can't be found, return `undefined`.
+   *
+   * @param moduleName The name of the module to check.
+   * @param symbolName The name of the symbol to look for.
+   */
+  getExportOfModule(moduleName: string, symbolName: string): ts.Symbol | undefined {
+    const moduleInfo = ts.resolveModuleName(
+      moduleName,
+      this.sourceFile.fileName,
+      this.languageServiceHost.getCompilationSettings(),
+      this.languageServiceHost as ts.ModuleResolutionHost
+    ).resolvedModule;
+
+    if (moduleInfo === undefined) {
+      // TypeScript couldn't resolve moduleName.
```

Comment:
Let's report, here. Above for the lit-* related symbols lets check and report again and ask "are you're importing lit-html or something imports it?" in the message.

Reply:
To continue our IRL chat -- I've been playing around with this. If a user only imports lit-html, they'll always get diagnostics a la "couldn't find lit-element", which would be confusing. We _also_ can't insert an ErrorExpression command at this point, since we don't have the concept of 'commands' until `convertFile`. 

Idea: what if we allow the lit-* related symbols to be undefined. Then later during `convertFile`, if we ever compare against an undefined lit-* symbol, we add diagnostics AND and ErrorExpression.

This would work well for users not importing lit-element: because `convertFile` would never run a code path that compares against the `property` symbol.

Thoughts?

Reply:
Ooh. Since we can generate AST error expressions once we start `convertFile`, perhaps the best thing to do would be to return an error expression in convertFile:

https://github.com/PolymerLabs/grimlock/blob/899c5da30443ed783a9c6c11b5bbddcbdf14ffb8/src/lib/source-file-converter.ts#L137-L150

If we find a soy compatible template function declaration and we can't resolve 'lit-html', then return an error expression. If we find a soy compatible class declaration and we can't resolve 'lit-element', return an error expression and report diagnostics.